Documentation/markdown/SetupHelp.md

```diff
@@ -50,7 +50,24 @@ These steps need to be completed before you can proceed with compiling OIP.
 	* Add this path to the environment variables. (Create new environment variable called `Boost_INCLUDE_DIR`. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. `C:\thirdparty\vs2017\x64\boost_1_65_1`).
 
 5. **Anaconda 2** (version with Python 2.7) - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe). 
+6. **PROJ 4:**
+
+	*	Download OSGeo4W64 Installer (find [here](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)).
+	*	In Installer select *Advanced Install*
+	*	Choose to install from Internet
+	*	Select directory, where PROJ 4 will be installed (`C:\OSGeo4W64` should be default option).
+	*	Select local package directory (e.g. `C:\local`)
```

Comment:
The local package directory could be in `OSGeo4W64\local`.

Reply:
Resolved: 21b0946